api/test/routes/game/answer_test.dart

```diff
@@ -127,7 +127,7 @@ void main() {
           final response = await route.onRequest(requestContext);
 
           expect(response.statusCode, HttpStatus.ok);
-          verifyNever(() => crosswordRepository.updateSolvedWordsCount());
+          verifyNever(() => crosswordRepository.updateSolvedWordsCount('id'));
```

Comment:
Let's check that is not called at all for any parameter
```suggestion
          verifyNever(() => crosswordRepository.updateSolvedWordsCount(any()));
```